.github/workflows/test.yml

```diff
@@ -8,6 +8,18 @@ on:
       - '**'
       - '!doc/**'
 jobs:
+  sanity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install goimports
+        run:  GO111MODULE=off go get golang.org/x/tools/cmd/goimports
```

Comment:
This is needed due to `go get ...` modifying the `[go.mod,go.sum]` files when attempting to grab this dependency: https://github.com/golang/go/issues/24250

I might've been able to get away with just restoring the changes made to those files, but seems unnecessarily complex.

Reply:
Not positive, but I think this gets easier with Go 1.16. Pretty sure you can do `go install <cmd>@<version>` without modifying anything in the project.

Reply:
Yeah, I saw a note about that in the 1.16 release notes, but I was under the impression that was only for the build/test commands. I ran this locally and was able to get `go install ...` working without having to mess around with that `GO111MODULE` environment variable. We'll see if that holds up in the next CI run.